pkgs/development/python-modules/groq/default.nix

```diff
@@ -0,0 +1,71 @@
+{
+  lib,
+  buildPythonPackage,
+  pytestCheckHook,
+  pytest-asyncio,
+  dirty-equals,
+  fetchPypi,
+  hatch-fancy-pypi-readme,
+  hatchling,
+  cached-property,
+  distro,
+  httpx,
+  pydantic,
+  sniffio,
+  anyio,
+  typing-extensions,
+}:
+
+buildPythonPackage rec {
+  pname = "groq";
+  version = "0.11.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-27mu/t84jd1IAex78+un9e22eUj+wM0oKdlyRAWfQqc=";
+  };
+
+  patches = [ ./ignore-warnings-pytest.patch ];
+
+  build-system = [
+    hatch-fancy-pypi-readme
+    hatchling
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-asyncio
+    dirty-equals
+  ];
+
+  pytestFlagsArray = [
+    "--ignore=tests/api_resources"
+    "--ignore=tests/test_client.py"
+  ];
```

Comment:
Please use `disabledTestPaths` instead.

Reply:
This was directly taken from docs, maybe we should change them

Reply:
Also, I left the client one in there in order to drop an unused dependency, but it's not technically a path so maybe I should continue to use the flag for (and only for) this